src/extension.ts

```diff
@@ -82,9 +87,12 @@ async function selectWorkItem() {
 			quickPick.onDidAccept(() => {
 				quickPick.hide();
 				for (const selectedItem of quickPick.selectedItems) {
-					getGitExtension().appendToCheckinMessage(`#${selectedItem.workItemId}`);
+					getGitExtension().appendToCheckinMessage(`#${selectedItem.workItemId}`, repo);
 				}
 			});
+		} else {
+			quickPick.hide();
+			vscode.window.showWarningMessage('Uanble to load work items.');
```

Comment:
typo

Reply:
Just wanted to test you 😜 nice catch. I'll do it tomorrow as I just shut down my laptop

Reply:
No hurry from my side. Thanks for taking the time!